source/_docs/locked_out.md

```diff
@@ -10,16 +10,33 @@ or need to recover your data.
 
 ### Home Assistant (including Supervised)
 
-If you are still logged in to the web interface with your user, then you are in luck. Add a new user as an administrator and give the new user a password you can remember. Then log out, and log in with this new user. You may then delete the old user account. But this way, your configuration will remain, and you don't have to do a new onboarding process.
+If you are still logged in to the web interface with your user, then you are in luck. Add a new user as an administrator and give the new user a password you can remember. Then log out, and log in with this new user. You may then reset your password via this new administrator account (and then delete this new account), or you can delete your old user account. Either way, your configuration will remain, and you don't have to do a new onboarding process.
 
-If you’ve forgotten your user, then deleting the files mentioned above will be necessary to start a new onboarding process.
+If you’ve forgotten your username, then deleting the files mentioned above will be necessary to start a new onboarding process.
 
-If you know the user, but not the password and you can access the [Home Assistant console](https://www.home-assistant.io/hassio/commandline/) and use the command below:
+If you know the username, but not the password and you can access the [Home Assistant console](https://www.home-assistant.io/hassio/commandline/) and use the command below:
 
 Connect a keyboard and monitor to your device.
 
 `auth reset --username existing_user --password new_password`
 
+#### To reset a user's password, as an administrator
```

Comment:
What is unclear right now, is is this paragraph needed if I did the auth reset listed above?

Maybe adding an additional introduction in this paragraph helps? Also, if listing steps, please use a type of list to make clear it is a step-by-step instruction that is given.